plugins/openshift/skills/generating-ovn-topology/scripts/check-permissions.py

```
@@ -0,0 +1,212 @@
+        ("create", "services", "Create services"),
+    ]
+
+    def __init__(self, kubeconfig: str):
+        self.kubeconfig = kubeconfig
+        self.write_perms_found = False
+        self.write_perms_list: List[str] = []
+
+    def check_kubectl_available(self) -> bool:
+        """Check if kubectl is available in PATH."""
+        try:
+            subprocess.run(
```

Ruff (0.14.3)

`subprocess` call: check for execution of untrusted input

(S603)